<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FotoWall</class>
 <widget class="QWidget" name="FotoWall">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true">FotoWall#QWidget
{
    background-color: rgb(59, 59, 59);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QFrame" name="topBar">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget#topBar
{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(255, 255, 255, 255), stop:0.500 rgba(232, 232, 232, 255), stop:0.501 rgba(202, 202, 202, 255), stop:1 rgba(237, 237, 237, 255)) ;
border: 1px solid rgb(162, 162, 162);
border-top: 0px;
border-bottom-left-radius: 6px;
border-bottom-right-radius: 6px;

}</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>4</number>
        </property>
        <property name="rightMargin">
         <number>8</number>
        </property>
        <property name="bottomMargin">
         <number>4</number>
        </property>
        <item>
         <widget class="QToolBar" name="contentToolbar">
          <property name="palette">
           <palette>
            <active>
             <colorrole role="Button">
              <brush brushstyle="SolidPattern">
               <color alpha="0">
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Base">
              <brush brushstyle="SolidPattern">
               <color alpha="0">
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Window">
              <brush brushstyle="SolidPattern">
               <color alpha="0">
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
            </active>
            <inactive>
             <colorrole role="Button">
              <brush brushstyle="SolidPattern">
               <color alpha="0">
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Base">
              <brush brushstyle="SolidPattern">
               <color alpha="0">
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Window">
              <brush brushstyle="SolidPattern">
               <color alpha="0">
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
            </inactive>
            <disabled>
             <colorrole role="Button">
              <brush brushstyle="SolidPattern">
               <color alpha="0">
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Base">
              <brush brushstyle="SolidPattern">
               <color alpha="0">
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Window">
              <brush brushstyle="SolidPattern">
               <color alpha="0">
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
            </disabled>
           </palette>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string>#contentToolbar
{
/* background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(255, 255, 255, 255), stop:0.500 rgba(232, 232, 232, 255), stop:0.501 rgba(202, 202, 202, 255), stop:1 rgba(237, 237, 237, 255)) ; */
    background-color: rgba(255, 255, 255, 0);
}</string>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="floatable">
           <bool>false</bool>
          </property>
          <addaction name="aAddPicture"/>
          <addaction name="aAddText"/>
          <addaction name="aAddVideo"/>
          <addaction name="aAddFlickr"/>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_3">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>28</height>
           </size>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Liberation Sans'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Taille :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="projectType">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <item>
           <property name="text">
            <string>FotoWall</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>CD Cover</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>DVD Cover</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Exact Size</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_2">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>28</height>
           </size>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="menu">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>28</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QToolButton
{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(250, 250, 250, 255), stop:0.500 rgba(200, 200, 200, 255), stop:0.501 rgba(202, 202, 202, 255), stop:1 rgba(230, 230, 230, 255)) ;
border: 1px solid rgb(144, 144, 144);
border-radius: 5px;
}</string>
          </property>
          <property name="text">
           <string>Menu</string>
          </property>
          <property name="icon">
           <iconset resource="fotowall.qrc">
            <normaloff>:/data/action-configure.png</normaloff>:/data/action-configure.png</iconset>
          </property>
          <property name="popupMode">
           <enum>QToolButton::InstantPopup</enum>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextBesideIcon</enum>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="spacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>17</width>
            <height>32</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>28</height>
           </size>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="loadButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>28</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QToolButton
{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(250, 250, 250, 255), stop:0.500 rgba(200, 200, 200, 255), stop:0.501 rgba(202, 202, 202, 255), stop:1 rgba(230, 230, 230, 255)) ;
border: 1px solid rgb(144, 144, 144);
border-radius: 5px;
}</string>
          </property>
          <property name="text">
           <string>load</string>
          </property>
          <property name="icon">
           <iconset resource="fotowall.qrc">
            <normaloff>:/data/action-snapshot.png</normaloff>:/data/action-snapshot.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextBesideIcon</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="saveButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>28</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QToolButton
{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(250, 250, 250, 255), stop:0.500 rgba(200, 200, 200, 255), stop:0.501 rgba(202, 202, 202, 255), stop:1 rgba(230, 230, 230, 255)) ;
border: 1px solid rgb(144, 144, 144);
border-radius: 5px;
}</string>
          </property>
          <property name="text">
           <string>save</string>
          </property>
          <property name="icon">
           <iconset resource="fotowall.qrc">
            <normaloff>:/data/action-save.png</normaloff>:/data/action-save.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextBesideIcon</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="exportButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>28</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QToolButton
{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(250, 250, 250, 255), stop:0.500 rgba(200, 200, 200, 255), stop:0.501 rgba(202, 202, 202, 255), stop:1 rgba(230, 230, 230, 255)) ;
border: 1px solid rgb(144, 144, 144);
border-radius: 5px;
}</string>
          </property>
          <property name="text">
           <string>export</string>
          </property>
          <property name="icon">
           <iconset resource="fotowall.qrc">
            <normaloff>:/data/fotowall.png</normaloff>:/data/fotowall.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextBesideIcon</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="quitButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>28</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QToolButton
{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(250, 250, 250, 255), stop:0.500 rgba(200, 200, 200, 255), stop:0.501 rgba(202, 202, 202, 255), stop:1 rgba(230, 230, 230, 255)) ;
border: 1px solid rgb(144, 144, 144);
border-radius: 5px;
}</string>
          </property>
          <property name="text">
           <string>quit</string>
          </property>
          <property name="icon">
           <iconset resource="fotowall.qrc">
            <normaloff>:/data/action-delete.png</normaloff>:/data/action-delete.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextBesideIcon</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="FWGraphicsView" name="view">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
    </widget>
   </item>
  </layout>
  <action name="aAddPicture">
   <property name="icon">
    <iconset resource="fotowall.qrc">
     <normaloff>:/data/insert-image.png</normaloff>:/data/insert-image.png</iconset>
   </property>
   <property name="text">
    <string>Add Picture</string>
   </property>
   <property name="shortcut">
    <string>P</string>
   </property>
  </action>
  <action name="aAddText">
   <property name="icon">
    <iconset resource="fotowall.qrc">
     <normaloff>:/data/insert-text.png</normaloff>:/data/insert-text.png</iconset>
   </property>
   <property name="text">
    <string>Add Text</string>
   </property>
   <property name="shortcut">
    <string>T</string>
   </property>
  </action>
  <action name="aAddVideo">
   <property name="icon">
    <iconset resource="fotowall.qrc">
     <normaloff>:/data/insert-camera.png</normaloff>:/data/insert-camera.png</iconset>
   </property>
   <property name="text">
    <string>Add Video</string>
   </property>
   <property name="shortcut">
    <string>V</string>
   </property>
  </action>
  <action name="aAddFlickr">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="fotowall.qrc">
     <normaloff>:/data/insert-download.png</normaloff>:/data/insert-download.png</iconset>
   </property>
   <property name="text">
    <string>Add Flickr</string>
   </property>
   <property name="shortcut">
    <string>F</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FWGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>FotoWall.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="fotowall.qrc"/>
 </resources>
 <connections/>
</ui>
